### PR TITLE
Added ViaQ/vector:v0.21-rh

### PR DESCRIPTION
**Pull request by vimalk78**
Added build for ViaQ/vector:v0.21-rh

### CODE REVIEW

File: core-services/image-mirroring/openshift-logging/mapping_logging_latest_quay [hunk 1]
```diff
@@ -8,4 +8,5 @@ registry.ci.openshift.org/logging/5.y:logging-elasticsearch6 quay.io/openshift-l
 registry.ci.openshift.org/logging/5.y:logging-eventrouter quay.io/openshift-logging/eventrouter:0.3
 registry.ci.openshift.org/logging/5.y:logging-kibana6 quay.io/openshift-logging/kibana6:6.8.1
 registry.ci.openshift.org/logging/5.y:log-file-metric-exporter quay.io/openshift-logging/log-file-metric-exporter:1.0
-registry.ci.openshift.org/logging/5.y:vector quay.io/openshift-logging/vector:0.14.1
\ No newline at end of file
+registry.ci.openshift.org/logging/5.y:vector quay.io/openshift-logging/vector:0.14.1
+registry.ci.openshift.org/logging/5.y:vector quay.io/openshift-logging/vector:0.21
```
[the remark this answers is not in the document]
Comment (vimalk78) [reply core-services/image-mirroring/openshift-logging/mapping_logging_latest_quay]: updated

File: ci-operator/config/ViaQ/vector/ViaQ-vector-v0.14-rh.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ images:
   from: ubi8
   to: vector-test-unit
 promotion:
-  name: 5.y
+  name: 0.14-rh
```
Comment (jcantrill) [inline ci-operator/config/ViaQ/vector/ViaQ-vector-v0.14-rh.yaml:19]: v0.14-rh

File: ci-operator/config/ViaQ/vector/ViaQ-vector-v0.21-rh.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    tag: golang-1.15
+images:
+- dockerfile_path: Dockerfile
+  from: ubi8
+  to: vector
+- dockerfile_path: Dockerfile.unit
+  from: ubi8
+  to: vector-test-unit
+promotion:
+  name: 0.21-rh
```
Comment (jcantrill) [inline ci-operator/config/ViaQ/vector/ViaQ-vector-v0.21-rh.yaml:19]: v0.21-rh

**Pull request merged**